CONTRIBUTING.md

```diff
@@ -24,40 +24,61 @@ Issues
 The easiest way to help out is to log any issues you find in the books. Unclear passages, errors of
 all kinds, even better ways to present something -- just go to the [issues page][].
 
-**Before creating a new issue**, please review existing issues to see if someone has already
-submitted the same one. Chances are you're not the first to encounter something, so a little quick
-research can save everyone some hassle. It's also a good idea to verify that problems still exist in
-the development branch (`dev`) when creating new issues.
+1. First ensure that the issue is still outstanding in the current development branch (`dev`). Often
+   the issue has already been addressed or no longer applies to the latest in-development version.
 
-When entering a new issue, please include all relevant information. For content issues, include the
-book or books this applies to, and specific locations that should be reviewed. Similarly for code:
-please include the file, function/class, and line number(s) if that applies.
+2. **Before creating a new issue**, please review existing issues to see if someone has already
+   submitted the same one. Chances are you're not the first to encounter something, so a little
+   quick research can save everyone some hassle. If you have new information, please continue the
+   thread in the existing issue.
 
-Finally, _please keep issues focused on a single problem or suggestion_. If discussion prompts you
-to think of another related issue, create a new issue for that topic.
+3. When entering a new issue, please include all relevant information. For content issues, include
+   the book or books this applies to, and specific locations that should be reviewed. Similarly for
+   code: please include the file, function/class, and line number(s) if that applies.
+
+4. Finally, _please keep issues focused on a single problem or suggestion_. If discussion prompts
+   you to think of related issue, create a _new_ issue for that topic, and add a link back to the
+   discussion or related issue (just use the "#NNN" syntax for issue/discussion/pull-request NNN --
```

Comment:
I would slightly rephrase this sentences: _If discussion prompts you to think of a related issue, create a _separate_ issue for that topic and link back to the original discussion or issue (just use the ..._

Reply:
I like it.